docs/api.md

```diff
@@ -274,6 +275,13 @@ await context.close();
 - [browserContext.setPermissions(origin, permissions[])](#browsercontextsetpermissionsorigin-permissions)
 <!-- GEN:stop -->
 
+#### event: 'close'
+
+Emitted when Browser context gets closed. This might happen because of one of the following:
+- Browser context is closed.
+- Browser application is closed or crashed.
+- The [`browser.disconnect`](#browserdisconnect) method was called.
```

Comment:
There is no such method anymore, it's called `browser.close` now.

Reply:
Done